expression/builtin_time.go

```diff
@@ -746,3 +751,52 @@ func parseDayInterval(sc *variable.StatementContext, value types.Datum) (int64,
 	}
 	return value.ToInt64(sc)
 }
+
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_unix-timestamp
+func builtinUnixTimestamp(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
+	if len(args) == 0 {
+		now := time.Now().Unix()
+		d.SetInt64(now)
+		return
+	}
+
+	d.SetInt64(0)
+	var (
+		t  types.Time
+		t1 time.Time
+		e  error
```

Comment:
I know, but if I use `if err != nil { return }` , this function would return error, which is not the same as mysql behavior.